### PR TITLE
LP-2627 - Add backend validations for name, designation and description

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
- Change max_length for name, designation and description
- Add help_text for name, image, designation, description
- Add single migration for all changes

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1624/workflows/workflows_id/jobs/1624/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/our_team/constants.py (100%)
openedx/adg/lms/our_team/models.py (100%)
-------------
Total:   8 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/our_team/models.py [hunk 1]
```diff
@@ -27,18 +27,32 @@ def board_of_trustees(self):
...
     """
     Model for team members and board of trustees in about page
     """
 
-    name = models.CharField(verbose_name=_('Name'), max_length=150, )
-    designation = models.CharField(verbose_name=_('Designation'), max_length=150, )
+    name = models.CharField(
+        verbose_name=_('Name'),
+        max_length=70,
+        help_text=_('Add a name of max {char_limit} characters').format(char_limit=MAX_NAME_LENGTH),
```
Comment (annas-usman) [inline openedx/adg/lms/our_team/models.py:39]: lets use f string here and in the help text of designation and description as well

File: openedx/adg/lms/our_team/models.py [hunk 2]
```diff
@@ -27,18 +27,32 @@ def board_of_trustees(self):
...
 class OurTeamMember(models.Model):
     """
     Model for team members and board of trustees in about page
     """
 
-    name = models.CharField(verbose_name=_('Name'), max_length=150, )
-    designation = models.CharField(verbose_name=_('Designation'), max_length=150, )
+    name = models.CharField(
+        verbose_name=_('Name'),
+        max_length=70,
```
Comment (asadali145) [inline openedx/adg/lms/our_team/models.py:38]: Can we use the defined constants here for the consistency?

**Review by annas-usman — CHANGES_REQUESTED** [review]

File: openedx/adg/lms/our_team/models.py (hunk 2 above)
Comment (asadali145) [reply openedx/adg/lms/our_team/models.py:38]: Same goes for the below fields.

File: openedx/adg/lms/our_team/models.py (hunk 1 above)
Comment (asadali145) [inline openedx/adg/lms/our_team/models.py:39]: Use f-strings for all.

**Review by asadali145 — CHANGES_REQUESTED** [review]

**Review by harris-irfan — APPROVED** [review]
Apart from the things already pointed out, LGTM.

File: openedx/adg/lms/our_team/models.py (hunk 1 above)
Comment (asadali145) [reply openedx/adg/lms/our_team/models.py:39]: f-strings can't be use for the translated strings. You can go with `format`.

Comment (MunamMubashir-arbisoft) [reply openedx/adg/lms/our_team/models.py:39]: Cannot use fstrings with translations.
https://code.djangoproject.com/ticket/30109

**Review by annas-usman — APPROVED** [review]

**Review by asadali145 — APPROVED** [review]
LGTM, Just make sure to revert the migration on dev server if it was already deployed on dev.

**Comment by asadali145** [comment]
Lets wait for the workflows as well.

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1626/workflows/workflows_id/jobs/1626/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/our_team/constants.py (100%)
openedx/adg/lms/our_team/models.py (100%)
-------------
Total:   8 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Pull request merged**